interface/wx/config.h

```diff
@@ -549,6 +549,51 @@ class wxConfigBase : public wxObject
     */
     bool Read(const wxString& key, long* l,
               long defaultVal) const;
+    /**
+        Reads a 64-bit long long value, returning @true if the value was found.
+        If the value was not found, @a ll is not changed.
+
+        @since 3.1.5
+
+        @beginWxPerlOnly
+        Not supported by wxPerl.
+        @endWxPerlOnly
+    */
+    bool Read(const wxString& key, wxLongLong_t* ll) const;
+    /**
+        Reads a 64-bit long long value, returning @true if the value was found.
+        If the value was not found, @a defaultVal is used instead.
+
+        @since 3.1.5
+
+        @beginWxPerlOnly
+        Not supported by wxPerl.
+        @endWxPerlOnly
+    */
+    bool Read(const wxString& key, wxLongLong_t* ll,
```

Comment:
The function declaration seems to be incomplete, should probably look like
```suggestion
    bool Read(const wxString& key, wxLongLong_t* ll, wxLongLong_t defaultVal) const;
```

Reply:
Oops, thanks for noticing!